pyuvsim/telescope.py

```diff
@@ -151,14 +136,7 @@ def __init__(
             else:
                 raise ValueError("Invalid beam list: " + str(beam_list))
 
-        # If any UVBeam objects are passed in, update uvb_params:
-        list_uvb_params = {}
-        if uvb_params is None:
-            uvb_params = {}
-            if not self.string_mode:
-                list_uvb_params = self._scrape_uvb_params(self._obj_beam_list, strict=False)
-        self.uvb_params.update(list_uvb_params)
-        self.uvb_params.update(uvb_params)    # Optional parameters for the UVBeam objects
+        # If any UVBeam objects are passed in, set uvb_read_kwargs:
```

Comment:
doesn't even look like we're checking for UVBeam objects at all. just the read_kwargs. Do we need this comment still?

Reply:
probably not.